packages/core/src/components/header/header-dropdown/header-dropdown.tsx

```diff
@@ -45,7 +45,7 @@ export class TdsHeaderDropdown {
 
   handleSlottedItemClick = (event: MouseEvent | KeyboardEvent) => {
     const eventSource = (event.target as HTMLElement).tagName.toLowerCase();
-    if (['a', 'button'].includes(eventSource)) {
+    if (['a', 'button', 'div'].includes(eventSource)) {
```

Comment:
I looked very briefly on this one, but I wonder, can we provide a solution where clicking on parent element would provide a better outcome? Ofc, we would need to check which elements and update a logic a bit...

```suggestion
    if (['tds-header-dropdown-list-item'].includes(eventSource)) {
```

Reply:
This change
`if (['tds-header-dropdown-list-item'].includes(eventSource)) {`
by itself doesn't work (I tested it), which I think is what you said. I will investigate if it's possible to fix a solution where clicking any element inside a parent element works

Reply:
The reason it doesn't work now is because the eventSource is one of 'a', 'button', 'div' when we click inside a dropdown row, even if they are child elements of 'tds-header-dropdown-list-item'